src/site/content/en/blog/lighthouse-whats-new-6.0/index.md

```diff
@@ -0,0 +1,473 @@
+  <img class="w-screenshot" src="lighthouse-ci-3.png" alt="Lighthouse CI." width="600">
+</figure>
+
+## Renamed Chrome DevTools panel {: #devtools }
+
+We've renamed the Audits panel to the Lighthouse panel. Enough said!
+
+Depending on your DevTools window size, the panel is probably behind the "»" button. You can drag
+the tab to change the order.
+
+To quickly reveal the panel with the [Command
+menu](https://developers.google.com/web/tools/chrome-devtools/command-menu): 
+
+1. Press {% Instruction 'devtools' %} to open DevTools.
+1. Press {% Instruction 'devtools-command', 'ol' %} to open the **Command** menu.
+1. Start typing "Lighthouse".
+1. Press `Enter`.
+
+## Mobile emulation {: #emulation }
+
+Lighthouse follows a mobile-first mindset. Performance problems are more apparent under typical
+mobile conditions, but developers often don't test in these conditions. This is why the default
+configuration in Lighthouse applies mobile emulation. The emulation consists of:
+
+-  Simulated slow network and CPU conditions (via a simulation engine called
+   [Lantern](https://github.com/GoogleChrome/lighthouse/blob/master/docs/lantern.md)).
+-  Device screen emulation (the same found in Chrome DevTools).
+
+Since the beginning, Lighthouse has used a Nexus 5X as its reference device. In recent years, most
+performance engineers have been using Moto G4 for testing purposes. Now Lighthouse is following suit
+and has changed its reference device to the Moto G4. In practice, this change is not very noticable,
+but here are all the changes detectable by a webpage:
+
+-  Screen size is changed from 412x660 px to 360x640 px.
+-  The useragent changes slightly, just the device portion (previously "`Nexus 5 Build/MRA58N`"
+   will now be `"Moto G (4)"`.
+
+As of Chrome 81, Moto G4 is available in the Chrome DevTools device emulation list.
+
+![image](lighthouse-motog4.png)
+
+## Browser extension {: #extension }
+
+The
+[Chrome extension for Lighthouse](https://chrome.google.com/webstore/detail/lighthouse/blipmdconlkpinefehnmjammfjpmpbjk)
+has been a convenient way to run Lighthouse locally. Unfortunately, it was complicated to support.
+We felt that because the Chrome DevTools Lighthouse panel is a better experience (the report
+integrates with other panels), we could reduce our engineering overhead by simplifying the Chrome
+extension.
+
+Instead of running Lighthouse locally, the extension now uses the [PageSpeed Insights
+API](https://developers.google.com/speed/docs/insights/v5/get-started). We recognize that this will
+not be a sufficient replacement for some of our users. These are the key differences:
+
+-  Pagespeed Insights is unable to audit non-public websites, since it is run via a remote
+   server and not your local Chrome instance. 
+   -  If you need to audit a non-public website, use the DevTools Lighthouse panel, or the
+      Node CLI.
+
+-  Pagespeed Insights is not guaranteed to use the latest Lighthouse version.
```

Comment:
carrying over the thread from the doc. this is still so weird to me.   looking at someone else's project i would not consider "latest version" to ever mean `master` branch.  but regardless, I imagine everyone would be fine if we say  "release" instead of "version" to deambiguify. 


how about

```md
- PageSpeed Insights may not be using the latest Lighthouse release.
   - In general, PageSpeed Insights will be updated to the latest Lighthouse release within 1-2 weeks. Use the Node CLI if you want to run a specific Lighthouse version.
```